src/arch/arm/imx/iomuxc/iomuxc.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ *
+ * @date Nov 28, 2018
+ * @author Anton Bondarev
+ */
+
+#include <util/log.h>
+
+#include <stdint.h>
+
+#include <hal/reg.h>
+#include <drivers/common/memory.h>
+
+#include <drivers/iomuxc.h>
+
+#include <framework/mod/options.h>
+
+#define IOMUXC_BASE            OPTION_GET(NUMBER, base_addr)
+
+void iomuxc_set_reg(uint32_t reg, uint32_t val) {
```

Comment:
I think `iomuxc_read()`/`iomuxc_write()` are better (and shorter) names for these functions.